test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -121,6 +121,7 @@ public void It_migrates_an_app_with_scripts_and_the_scripts_run()
         private MigratedBuildComparisonData GetDotnetNewComparisonData(string projectDirectory, string dotnetNewType)
         {
             DotnetNew(projectDirectory, dotnetNewType);
+            File.Copy("NuGet.tempaspnetpatch.config", Path.Combine(projectDirectory, "NuGet.Config"));
```

Comment:
Im surprised this doesn't just use the nuget.config from the repo root by default. LGTM though